dev_sandbox/DataExploration_1D.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f00cf5bf",
+   "metadata": {},
+   "source": [
+    "# Milestone 1: Data Exploration\n",
+    "### Install Hints\n",
+    "Make sure to install your conda environment using the doc/envs/pydrop_dev.yml file for the most consistent results\n",
+    "### Goals\n",
+    "The goals of this notebook are to:\n",
+    "1. Begin plotting the data from a single channel and visualize the distribution and spread of samples. \n",
+    "2. Obtain basic descriptive statistics and attempt to fit a normal distribution to gauge spread.\n",
+    "3. Code an initial clustering proof-of-concept to begin modeling progress. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "7bdae44b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
```

Comment:
suggestion(non-blocking): it's not required, but it's tidier to alphabetize your imports.